UIKit/Tasks/ViewController.swift

```diff
@@ -3,10 +3,157 @@
+
+    private let pictureBackgroundView: UIImageView = {
+        let pictureBackgroundView = UIImageView()
+        pictureBackgroundView.image = UIImage(named: "picture background")
+        pictureBackgroundView.contentMode = .scaleAspectFill
+        pictureBackgroundView.frame = CGRect(x: 0, y: 50, width: 380, height: 760)
+        return pictureBackgroundView
+    }()
+
+    private let nameLabel: UILabel = {
+        let nameLabel = UILabel()
+        nameLabel.backgroundColor = .colorWelcom
+        nameLabel.frame = CGRect(x: 0, y: 49, width: 380, height: 82)
+        nameLabel.textAlignment = .center
+        nameLabel.numberOfLines = 0
+        nameLabel.font = .boldSystemFont(ofSize: 30)
+        nameLabel.textColor = .white
+        return nameLabel
+    }()
+
+    private lazy var additionbutton: UIButton = {
+        let additionbutton = UIButton()
+        additionbutton.frame = CGRect(x: 132, y: 507, width: 200, height: 200)
+        additionbutton.backgroundColor = .additionbutton
+        additionbutton.setTitle("Калькулятор", for: .normal)
+        additionbutton.titleLabel?.textAlignment = .center
+        additionbutton.titleLabel?.font = UIFont(name: "Verdana", size: 20)
+        additionbutton.layer.cornerRadius = 30
+        additionbutton.layer.borderWidth = 3
+        additionbutton.layer.borderColor = UIColor(named: "calculator button frame")?.cgColor
+        additionbutton.addTarget(self, action: #selector(calculateResult), for: .touchUpInside)
+        return additionbutton
+    }()
+
+    private lazy var guessNumberbutton: UIButton = {
+        let guessNumberbutton = UIButton()
+        guessNumberbutton.frame = CGRect(x: 82, y: 301, width: 150, height: 150)
+        guessNumberbutton.backgroundColor = .guessNumberbutton
+        guessNumberbutton.setTitle("Угадай число", for: .normal)
+        guessNumberbutton.titleLabel?.numberOfLines = 0
+        guessNumberbutton.titleLabel?.textAlignment = .center
+        guessNumberbutton.titleLabel?.font = UIFont.systemFont(ofSize: 25)
+        guessNumberbutton.layer.cornerRadius = 30
+        guessNumberbutton.layer.borderWidth = 3
+        guessNumberbutton.layer.borderColor = UIColor(named: "guess number button")?.cgColor
+        guessNumberbutton.addTarget(self, action: #selector(guessNumber), for: .touchUpInside)
+        return guessNumberbutton
+    }()
+
+    // MARK: - Life Cycle
+
+    override func viewDidAppear(_ animated: Bool) {
+        view.backgroundColor = .white
+        welcomeAlert()
+    }
+
     override func viewDidLoad() {
         super.viewDidLoad()
-        view.backgroundColor = .white
+        nameLabel.isHidden = true
+        setup()
+    }
+
+    // MARK: - Private Methods
+
+    @objc private func calculateResult() {
+        let alert = UIAlertController(title: "Введите ваши числа", message: nil, preferredStyle: .alert)
+        let alertResult = UIAlertController(title: "Ваш результат", message: "", preferredStyle: .alert)
```

Comment:
alertResult плохое имя, я подумала, что это какой-то результат. который будет возвращаться алертом
resultAlert наоборот делает акцент на том, что в переменной именно алерт, а не резалт
```suggestion
        let resultAlert = UIAlertController(title: "Ваш результат", message: "", preferredStyle: .alert)
```